subprojects/native/src/integTest/groovy/org/gradle/internal/nativeintegration/jansi/JansiLibraryFactoryIntegrationTest.groovy

```diff
@@ -1,61 +0,0 @@
-/*
- * Copyright 2016 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.gradle.internal.nativeintegration.jansi
-
-import org.gradle.util.Requires
-import spock.lang.Specification
-
-import static org.gradle.internal.nativeintegration.jansi.JansiLibraryFactory.*
-import static org.gradle.util.TestPrecondition.*
-
-class JansiLibraryFactoryIntegrationTest extends Specification {
```

Comment:
Why did you remove this test without replacement?

Reply:
This was not an integration tests in the sense that it actually ran Gradle. So I wanted to move this into `test` as I thought it would be better placed there. To avoid this to be executed several times on CI. But then I saw that the followin test is (almost) the same so I removed this one as "duplicate".

https://github.com/gradle/gradle/blob/master/subprojects/native/src/test/groovy/org/gradle/internal/nativeintegration/jansi/JansiLibraryFactoryTest.groovy

But rechecking now, I see that there is a slight difference in the tests. So I should probably add this back. Thanks for asking.

Would you agree that it should go into `test` then (at least for now until we might add another set for "normal integration tests")?

Reply:
Running this as a normal test is perfectly fine.

Reply:
https://github.com/gradle/gradle/commit/a2cc431d4165eb53338a6e6cbe5db43aead2e3d1